docs/mk-docs-setup.md

```diff
@@ -0,0 +1,42 @@
+# HMC MKdocs Setup
+
+## Project layout
+
+    mkdocs.yml    # The configuration file.
+    docs/
+        index.md  # The documentation homepage.
+        stylesheets  # CSS stylesheets to control look and feel
+        assets  # Images and other served material
+        ...       # Other markdown pages, images and other files.
+
+
+## Setting up MKdocs and dependancies
```

Comment:
It might be nice to just offer a container image for this and have that run.  Doesn't need to be in this PR.

Reply:
agree, would be nicer. created issue: https://github.com/Mirantis/hmc/issues/351

Reply:
This should be in the container and as a Make target ;)
So we could just publish it automatically in the GH pages